src/components/FilterPanel.jsx

```diff
@@ -73,9 +93,43 @@ export default function FilterPanel({ onClose }) {
             <Typography>{name}</Typography>
           </AccordionSummary>
 
-          <AccordionDetails sx={{ m: 1 }}>{filter}</AccordionDetails>
+          <AccordionDetails sx={{ m: 1 }}>{filter.comp}</AccordionDetails>
         </Accordion>
       ))}
+
+      {filtersChanged && (
+        <Box
+          sx={{
+            display: 'flex',
+            alignItems: 'center',
+            justifyContent: 'center',
+            mt: 2,
+          }}>
+          <WarningAmberOutlinedIcon color="warning" />
+          <Typography>Your changes have not yet been applied</Typography>
+        </Box>
+      )}
+
+      <Toolbar sx={{ justifyContent: 'space-evenly', mt: 2 }}>
+        <Button
+          variant="contained"
+          disabled={!filtersChanged}
+          onClick={() => {
+            setQueryParams({ minAmount, maxAmount, grades, majors });
+            onClose();
+          }}>
+          Apply
+        </Button>
+        <Button
```

Comment:
Hmm I know we talked about keeping this always enabled for mobile but I'm noticing that on desktop it doesn't make sense as it doesn't do anything if there are no pending changes. Maybe disable this one too on `!filtersChanged`?